Zend/zend_constants.stub.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/** @generate-class-entries */
+
+/** @cname E_ERROR */
+const E_ERROR = 1;
+/** @cname E_WARNING */
+const E_WARNING = 2;
```

Comment:
Thee issue here is that the values are also exposed in the manual, and displaying `1<<x` there would be a bit weird. Maybe we could make the intent (no overlap of bits) more clear by some comment, but I'm not sure it's needed. Or do you think it is?

Reply:
Another possibility I can come up with is to go with bit-shifting/expressions, and try to evaluate them when the manual is generated. We should be fine If there is no unknown constant in the expression. However, this would require some additional work, and I'd like to address any issue which is not crutial/trivial to fix in a followup, as this PR is already way too huge.

Reply:
If the manual prefers avoiding bit-shifts then that's completely fine by me.